components/automate-ui/src/app/entities/projects/project.reducer.ts

```diff
@@ -79,7 +79,7 @@ export function projectEntityReducer(
 
     case ProjectActionTypes.GET_SUCCESS:
       return set(GET_STATUS, EntityStatus.loadingSuccess,
-        projectEntityAdapter.addOne(action.payload.project, state));
+        projectEntityAdapter.upsertOne(action.payload.project, state));
```

Comment:
If you use addOne, it doesn't create a new object and thus doesn't trigger ngrx store change detection. That means that any select that is focused on this object (like `projectFromRoute`) won't actually get updates from the store. TL;DR I think we should be using `upsertOne` everywhere we are using `addOne` (not just on create like my tech debt card proposes https://github.com/chef/automate/issues/1199).

Reply:
good catch! we should probably update that tech debt card